docs/user/FlowVariables.md

```diff
@@ -0,0 +1,268 @@
+| `MAKE_TRACKS`         | Tcl file that defines add routing tracks to a floorplan.                                                                                                                         |
+| `IO_PIN_MARGIN`       | Maximum number of I/O pins placed outside floorplan boundary. Integer value.                                                                                                     |
+| `IO_PLACER_H`         | The metal layer on which to place the I/O pins horizontally (top and bottom of the die).                                                                                         |
+| `IO_PLACER_V`         | The metal layer on which to place the I/O pins vertically (sides of the die).                                                                                                    |
+
+
+### Placement
+
+
+| Variable                             | Description                                                                                                                                   |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| `CELL_PAD_IN_SITES_GLOBAL_PLACEMENT` | Cell padding on both sides in site widths to ease routability during global placement. Default 2 sites.                                       |
+| `CELL_PAD_IN_SITES_DETAIL_PLACEMENT` | Cell padding on both sides in site widths to ease routability in detail placement. Default 1 sites.                                           |
+| `MAX_WIRE_LENGTH`                    | Specifies the maximum wire length used by the resizer to reduce RC delay by inserting buffers. Integer value.                                 |
+| `WIRE_RC_LAYER`                      | Metal Layer to use for parasitics estimations.                                                                                                |
+| `CELL_PAD_IN_SITES`                  | Cell padding in SITE widths to ease rout-ability. Integer value.                                                                              |
+| `PLACE_DENSITY`                      | The desired placement density of cells. It reflects how spread the cells would be on the core area. 1.0 = closely dense. 0.0 = widely spread. |
+
+
+### Clock Tree Synthesis(CTS)
+
+
+| Variable       | Description                                                                                                   |
+|----------------|---------------------------------------------------------------------------------------------------------------|
+| `CTS_BUF_CELL` | The buffer cell used in the clock tree.                                                                       |
+| `FILL_CELLS`   | Filler cells list used to fill any spaces between regular library cells to avoid planarity problems post-CTS. |

```

Comment:
Fill cells are used to fill empty sites and don't relate to planarity - that is metal fill.